plugins/tangle/tangle_processor.go

```diff
@@ -112,20 +116,20 @@ func processIncomingTx(incomingTx *hornet.Transaction, request *rqueue.Request,
 	// Release shouldn't be forced, to cache the latest transactions
 	defer cachedTx.Release(!isNodeSyncedWithThreshold) // tx -1
 
+	// since we only add the approvees if there was a source request, we only
+	// request them for transactions which should be part of milestone cones
+	if request != nil {
```

Comment:
Move that function back into the `!alreadyAdded` condition, since this works now with the fix in rqueue for me.
(but I only tested with my `processing` map branch)

Reply:
moved